spec/renderer/renderer_spec.rb

```diff
@@ -0,0 +1,178 @@
+require 'spec_helper'
+
+RSpec.describe 'simple rendering' do
+  let(:user1) { User.new(id: 1, name: 'dhh', email: 'dhh@rails.com') }
+  let(:user2) { User.new(id: 2, name: 'wycats', email: 'wycats@ember.com') }
+
+  let(:tweet) { Tweet.new(content: "Tweet", retweets: retweet) }
+  let(:retweet) { Tweet.new(author: user2, parent: tweet) }
+
+  let(:resource_klass) { Class.new(UserResource) }
+  let(:resource) { resource_klass.new(user: user1) }
+  let(:options) { {} }
+
+  let(:rendered) do
+    JSONAPI.render(resource, options)
+  end
+
+  let!(:now) { Time.now }
+
+  before do
+    allow(Time).to receive(:now) { now }
+  end
+
+  it 'renders valid jsonapi' do
+    expect(rendered).to eq({
+      data: {
+        id: '1',
+        type: 'users',
+        attributes: {
+          name: 'dhh',
+          email: 'dhh@rails.com'
+        },
+        relationships: {
+          tweets: {},
+          friends: {},
+          messages: {}
```

Comment:
Yeah I agree, I just fixed this in AMS actually. This is just current gem behavior. My original intent was to check in a passing suite, then fix it.